pkg/console/operator/operator.go

```diff
@@ -251,3 +260,21 @@ func (c *consoleOperator) removeConsole(cr *operatorsv1.Console) error {
 
 	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
 }
+
+func (c *consoleOperator) handleConfigMapSync(operatorConfig *operatorsv1.Console) error {
+	// Get Operator Config here
+	logoName := operatorConfig.Spec.Customization.CustomLogoFile.Name
+	// add syncing for the custom logo config map
+	if logoName != "" {
+		err := c.resourceSyncer.SyncConfigMap(
+			resourcesynccontroller.ResourceLocation{Namespace: api.OpenShiftConsoleNamespace, Name: logoName},
```

Comment:
The destination must be static.

Reply:
I wish I could make the destination name static but that value is set by the user.  The SyncConfigMap does not have an issue with me passing a dynamic name into the ResourceLocations.

Reply:
Try `oc apply -f` on this `operator config`:

```yaml
apiVersion: operator.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec:
  customization:
    customProductName: Muh Clusta
    customLogoFile:
      name: console-config  # uh oh!  We gonna blow up....
      key: fake-logo.png
```

Reply:
I am glad you changed your mind on having this dynamic.  I will set the destination name to a set name and that will prevent existing configmaps from being clobbered.

Reply:
done

Reply:
>  I am glad you changed your mind on having this dynamic. 

I dont know what this means?

Reply:
We haven't changed our minds.  The user SHOULD be able to do:

```yaml
    customLogoFile:
      name: console-config  # or service-ca or whatever they want to call it, we don't care...
      key: fake-logo.png
```
And it SHOULD NOT destroy `console-config`.  This should be copied into a `configmap` named `custom-logo` in `openshift-console` with no risk of destroying any console config.